installers/legacy-install.ps1

```diff
@@ -30,6 +30,8 @@ Set-StrictMode -Off
 
 # ignore project file if we are already in an activated environment
 $Env:ACTIVESTATE_PROJECT=""
+# disable auto-updates to prevent that the State Tool automatically updates to the latest version
+$Env:ACTIVESTATE_CLI_DISABLE_UPDATES = "true"
```

Comment:
Indeed weird.  But, in my local tests, it got updated to the latest version...  That might have been because I didn't run `state clean uninstall`, such that the config with the timestamp still existed from an earlier installation of that channel.